### ISSUE TITLE
`timestamp` column shows `None` if value is `0`

### ISSUE THREAD

**Issue by georgexsh** [dbcli/mycli#285]
Original mysql command tool show `0000-00-00 00:00:00`, but mycli shows `None` (vertical mode) or `<null>` (horizontal mode).
It has confused me for a while because that column is not nullable.

**Comment by mdsrosa** [comment]
@georgexsh Thank you for using mycli. But in this case I was unable to reproduce this behavior `0000-00-00 00:00:00` in mysql command tool. Could you please tell me which version of `MySQL` and `mycli` are you using?

**Comment by georgexsh** [comment]
`mysql  Ver 14.14 Distrib 5.5.47` and `mycli Version: 1.7.1`

**Comment by georgexsh** [comment]
```
mysql> desc ts_null;
+-------+-----------+------+-----+-------------------+-----------------------------+
| Field | Type      | Null | Key | Default           | Extra                       |
+-------+-----------+------+-----+-------------------+-----------------------------+
| c     | timestamp | NO   |     | CURRENT_TIMESTAMP | on update CURRENT_TIMESTAMP |
+-------+-----------+------+-----+-------------------+-----------------------------+
1 row in set (0.00 sec)

mysql> insert ts_null values (0);
Query OK, 1 row affected (0.01 sec)

mysql> select * from ts_null;
+---------------------+
| c                   |
+---------------------+
| 0000-00-00 00:00:00 |
+---------------------+
1 row in set (0.00 sec)
```
vs
```
mysql root@localhost:test> select * from ts_null;
+--------+
|      c |
|--------|
| <null> |
+--------+
1 row in set
Time: 0.001s

mysql root@localhost:test> select * from ts_null\G
***************************[ 1. row ]***************************
c | None

1 row in set
Time: 0.000s
```

**Comment by amjith** [comment]
I'm able to reproduce it, I'll see if I can take a stab at the fix.

**Comment by georgexsh** [comment]
@amjith thanks very much

**Comment by amjith** [comment]
This is now fixed in version 1.8.0. Please upgrade mycli: 

```
pip install -U mycli
```
